src/map/script.c

```diff
@@ -10861,6 +10861,91 @@ BUILDIN(addtimercount)
 	return true;
 }
 
+BUILDIN(gettimer)
+{
+	const char *event;
```

Comment:
perhaps provide a default value? according to Travis CI build, it generate warning since the variable might not be initialized if the script command doesn't provide the parameters